easybuild/easyconfigs/c/ctffind/ctffind-4.1.10-fosscuda-2018b.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Author: Ake Sandgren, HPC2N, Umea University
+
+easyblock = 'ConfigureMake'
+
+name = 'ctffind'
+version = '4.1.10'
+
+homepage = 'http://grigoriefflab.janelia.org/ctffind4'
+description = """Program for finding CTFs of electron micrographs."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
```

Comment:
Should probably drop this one down to GCC level (not GCCcore since it's performance critical and should be compiled by intel if using that toolchain)

Reply:
Agree

Reply:
Can't drop to GCC, must stay at foss(cuda), it needs FFTW.